src/main/java/org/folio/rest/impl/FundDistributionsValidationHelper.java

```diff
@@ -43,7 +43,10 @@ public Future<Void> validateFundDistributions(ValidateFundDistributionsRequest r
         List<FundDistribution> fundDistributionList = request.getFundDistribution();
         if (CollectionUtils.isNotEmpty(request.getAdjustments())) {
           if (validator.isAdjustmentIdsNotUnique(request.getAdjustments())) {
-            throw new HttpException(400, ADJUSTMENT_IDS_NOT_UNIQUE);
+            var parameter = new Parameter().withKey("adjustments").withValue(request.getAdjustments().toString());
+            var error = ADJUSTMENT_IDS_NOT_UNIQUE.toError().withParameters(List.of(parameter));
+            logger.error(JsonObject.mapFrom(error).encodePrettily());
```

Comment:
better to add some message before encoding error, in this case it will be easier to find error in the logs by matching with  phrase from source code

Reply:
and please do it for other places

Reply:
changed, error messages to meaningful one